git-cheatsheet/lang/pt.json

```diff
@@ -0,0 +1,195 @@
+      "docs": "Traz as modificações do repositório remoto para a branch local atual. No modo padrão, `git pull` é uma combinação de `git fetch` seguido de `git merge`."
+    },
+    "reset --hard x/x": {
+      "cmd": "reset --hard <remote>/<branch>",
+      "docs": "Reseta a branch local para ser igual à branch remota. Use `--hard origin/main` para descartar todos os commits feitos na branch local. AVISO: use este comando apenas para reiniciar um conflito de merge falho."
+    },
+    "fetch x x": {
+      "cmd": "fetch <remote> <refspec>",
+      "docs": "Realiza download dos objetos e referências do repositório remoto."
+    },
+    "push": {
+      "cmd": "push",
+      "docs": "Envia ao servidor remoto os commits em todas as branches que são **COMUNS** entre a máquina local e o servidor. As branches locais que não estão no servidor não são compartilhadas."
+    },
+    "push x x": {
+      "cmd": "push <remote> <branch>",
+      "docs": "Envia uma branch local nova (ou existente) para o servidor remoto."
+    },
+    "push x x:x": {
+      "cmd": "push <remote> <local>:<nome>",
+      "docs": "Envia uma branch local ao servidor com um nome diferente. <local> é o nome da branch local, e <nome> é o nome da branch no repositório remoto."
+    },
+    "branch -r": {
+      "cmd": "branch -r",
+      "docs": "Lista todas as branches remotas."
+    },
+    "push --delete": {
+      "cmd": "push <remote> --delete <branch>",
+      "docs": "Remove uma branch remota."
+    },
+    "clean": {
+      "cmd": "clean",
+      "docs": "Limpa o diretório atual removendo de forma recursiva arquivos não rastreados pelo git (untracked), começando do diretório atual. Use `-n` para verificar previamente quais arquivos seriam deletados. Use `-f` para deletá-los."
+    },
+    "stash push": {
+      "cmd": "stash push [<mensagem>]",
+      "docs": "Salva as suas modificações locais em uma área temporária (stash) e rode `git reset --hard` para desfazê-las. A <mensagem> é opcional e fornece a descrição do que foi salvo. Para salvar de forma mais rápida, você pode omitir o `push` e <mensagem>."
+    },
+    "stash apply": {
+      "cmd": "stash apply [<stash>]",
+      "docs": "Traz de volta as modificações da área temporária (stash) para a máquina local (workspace). O padrão é trazer a última modificação salva. A modificação continua salva na área temporária"
+    },
+    "stash pop": {
+      "cmd": "stash pop",
+      "docs": "Traz de volta as modificações da área temporária (stash) para a máquina local (workspace). O padrão é trazer a última modificação salva. A modificação é removida da área temporária"
+    },
+    "stash list": {
+      "cmd": "stash list",
+      "docs": "Lista as modificações salvas na área temporária (stash)."
+    },
+    "stash show": {
+      "cmd": "stash show [<stash>]",
+      "docs": "Show the changes recorded in the stash as a diff between the stashed state and its original parent. When no <stash> is given, shows the latest one."
```

Comment:
Are this and the next item intentionally not translated?

Reply:
I've forgotten this one. Will commit covering this =)